### ISSUE TITLE
Fix back button exiting app

### ISSUE THREAD

**Issue by maelchiotti** [maelchiotti/LocalMaterialNotes#293]
- Instead of closing the drawer
- Instead of exiting the selection mode

**Comment by NripPrajapat** [comment]
It works fine, when I press back on the bin page, it takes me to the app's home screen (main notes list), but it still won't work on selection menu nor from the label pages.

**Comment by maelchiotti** [comment]
Yes, I have trouble understanding why it won't work with the selection menu. Will look deeper into it.

However it works for me for the labels pages 🤔

**Comment by maelchiotti** [comment]
Oh well no it doesn't sometimes, my bad. A never ending battle 🥲
